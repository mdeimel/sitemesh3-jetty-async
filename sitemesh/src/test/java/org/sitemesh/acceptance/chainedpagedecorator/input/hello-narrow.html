<html>
  <head>
    <title>Hello Narrow</title>
  </head>
  <body>

   <p>Very simple document. Should appread in narrow layout.</p>

  </body>
</html>