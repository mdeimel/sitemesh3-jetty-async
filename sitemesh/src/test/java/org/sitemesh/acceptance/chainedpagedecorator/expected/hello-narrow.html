<html>
  <head>
    <title>Hello Narrow</title>
    <style type="text/css">
      .sidebar { float: left; width: 200px; }
      .narrow { width: 800px; }
    </style>
    <style type="text/css">
      .main { background-color: #eee; }
      body { font-family: arial,serif; }
    </style>
  </head>
  <body>

    <h1>Hello Narrow</h1>

    <div class="main">
      <div class="sidebar">
        <ul>
          <li>Some</li>
          <li>Additional</li>
          <li>Stuff</li>
        </ul>
      </div>

      <div class="narrow">
        <p>Very simple document. Should appread in narrow layout.</p>
      </div>
    </div>

  </body>
</html>